CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 1.4.2 - 2023-12-04
```

Comment:
should probably be removed as this will be added once a release is actually drafted as far as I know

Reply:
Yeah wasn't sure if I needed to add it, since it's a checkpoint in the issue template

Reply:
the line below is good IMO, I would simply remove the version and the date

Reply:
thanks, yes that was the correct way to do it.